playbooks/aws/openshift-cluster/tasks/launch_instances.yml

```diff
@@ -4,7 +4,9 @@
     docker_vol_ephemeral: "{{ lookup('env', 'os_docker_vol_ephemeral') | default(false, true) }}"
     env: "{{ cluster }}"
     env_host_type: "{{ cluster }}-openshift-{{ type }}"
+    env_node_type: "{{ cluster }}-openshift-{{ g_node_type }}"
```

Comment:
By "host pattern" I mean this (pattern expressions):  http://docs.ansible.com/ansible/intro_patterns.html